### PR TITLE
Add Java 11 build

### PR DESCRIPTION
**Pull request by bitwiseman**
@jenkinsci/java11-support 
Let's see what happens.

### CODE REVIEW

**Comment by bitwiseman** [comment]
Aw, I can't do this because I'm untrusted.  Still only building java8.

**Comment by dwnusbaum** [comment]
> Aw, I can't do this because I'm untrusted. Still only building java8.

You have [release permission](https://github.com/jenkins-infra/repository-permissions-updater/blob/acf08242fbff011301ac6e8a17217999df7690f2/permissions/plugin-workflow-cps.yml#L14), just need to track someone down with voice on IRC to get GitHub permission as well.

**Comment by bitwiseman** [comment]
Hm, I wonder why I didn't see those locally.

**Comment by bitwiseman** [comment]
@dwnusbaum Success!

File: src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition.java [hunk 1]
```diff
@@ -92,6 +92,11 @@ public boolean isLightweight() {
         this.lightweight = lightweight;
     }
 
+    @SuppressFBWarnings(
+            value = {"NP_LOAD_OF_KNOWN_NULL_VALUE", "RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE",
+                    "RCN_REDUNDANT_NULLCHECK_OF_NULL_VALUE", "RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE"},
+            justification = "false positives for try-resource in java 11"
```
Comment (dwnusbaum) [inline src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition.java]: Ah, this one is super annoying.

Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition.java]: This looks familiar; do you have a link to the SpotBugs issue offhand?

Comment (dwnusbaum) [reply src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition.java]: https://github.com/spotbugs/spotbugs/issues/756

**Pull request merged**